servers/xr_server.cpp

```diff
@@ -50,15 +50,15 @@ XRServer *XRServer::singleton = nullptr;
 
 XRServer *XRServer::get_singleton() {
 	return singleton;
-};
+}
 
 void XRServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("get_world_scale"), &XRServer::get_world_scale);
 	ClassDB::bind_method(D_METHOD("set_world_scale", "scale"), &XRServer::set_world_scale);
 	ClassDB::bind_method(D_METHOD("get_world_origin"), &XRServer::get_world_origin);
 	ClassDB::bind_method(D_METHOD("set_world_origin", "world_origin"), &XRServer::set_world_origin);
 	ClassDB::bind_method(D_METHOD("get_reference_frame"), &XRServer::get_reference_frame);
-	ClassDB::bind_method(D_METHOD("clear_reference_frame"), &XRServer::get_reference_frame);
+	ClassDB::bind_method(D_METHOD("clear_reference_frame"), &XRServer::clear_reference_frame);
```

Comment:
**Note** this is a bug that we missed for ages, it was pointing to the wrong method. This obviously breaks ABI, I'm assuming it is fine to add this as an exception?

Reply:
Personally, I think an exception for this would be fine, since it was broken previously

Reply:
Interestingly the CI isn't complaining about it..

Reply:
It's pretty weird that the CI isn't complaining indeed. The method hash should have changed.